bin/citrea/provers/risc0/guest-bitcoin/Cargo.toml

```diff
@@ -8,6 +8,7 @@ resolver = "2"
 
 [dependencies]
 # forcing cargo for this version
+alloy-primitives = { version = "0.7.6", default-features = false }
```

Comment:
Can we not use `workspace = true, default-features = false` here?

Reply:
guests are not part of the workspace afaik